README.md

```diff
@@ -168,7 +168,7 @@ On a production system, you likely want to start your Plug pipeline under your a
 $ mix new my_app --sup
 ```
 
-and then update `lib/my_app/application.ex` as follows:
+and then update your application file (for example, `lib/my_app/application.ex`) as follows:
```

Comment:
Won't it be strange that the paragraph is about creating a new project with `mix new my_app --sup`, and then say "for example"?

In order to say "for example", I believe the explanation should be a bit more generic, less tied to the command.

Reply:
I had missed that we just created the project with `--sup`! I will revert, sorry :).